extensions/control-plane/api/management-api/asset-api/src/main/java/org/eclipse/edc/connector/controlplane/api/management/asset/v3/AssetApi.java

```diff
@@ -50,10 +49,9 @@ public interface AssetApi {
                     @ApiResponse(responseCode = "409", description = "Could not create asset, because an asset with that ID already exists",
                             content = @Content(array = @ArraySchema(schema = @Schema(implementation = ApiCoreSchema.ApiErrorDetailSchema.class)))) }
     )
-    JsonObject createAsset(JsonObject asset);
+    JsonObject createAssetV3(JsonObject asset);
 
     @Operation(description = "Request all assets according to a particular query",
-            operationId = "requestAssetV3",
```

Comment:
by default the operationId reflects the method name, if the method is already called `requestAssetV3`, explicitly defining the `operationId` is not necessary

Reply:
but doesn't that mean, that if an `operationId` is present, it would override whatever the method name is?

Reply:
it is necessary to avoid the generated openapi mess (I debugged it, not sure about the root cause, but the rename thing definitely fixes the issue).
I see two appreciated side effects:
- it will avoid outdated `operationId` definitions (because they are sterile strings)
- defining different method names for different versions makes total sense (think about a method that in 2 different versions has 2 different signatures/behaviors)

Reply:
so long as it fixes the issue, i'm all for it!